host-contracts/contracts/InputVerifier.sol

```diff
@@ -211,7 +213,9 @@ contract InputVerifier is UUPSUpgradeableEmptyProxy, Ownable2StepUpgradeable, EI
 
             /// @dev This checks in particular that the list is non-empty.
             if (numHandles <= indexHandle || indexHandle > 254) revert InvalidIndex();
-            if (inputProofLen != 2 + 32 * numHandles + 65 * numSigners) revert DeserializingInputProofFail();
+
+            /// @dev Check that the inputProof is long enough to contain at least the numHandles + numSigners + handles + coprocessorSignatures
+            if (inputProofLen < 2 + 32 * numHandles + 65 * numSigners) revert DeserializingInputProofFail();
```

Comment:
Just to double-check the impact: is this correct to make this change in the evaluation of the `inputProofLen`? I mean, now that we have "remaining" bytes in the `inputProof` for `extraData`, we should check `inputProofLen < minimum expected size` instead of `!=`.

cc @immortal-tofu